dev/src/dev/toucan2_monitor.clj

```diff
@@ -56,19 +56,20 @@
    #'t2.pipeline/transduce-execute-with-connection
    :around
    :default
-   ::tracking))
+   ::monitor))
 
-(defn total-execution-time-ms
-  "Get the total execution time of all queries in ms."
+(defn summary
+  "Get the total number of queries and total execution time in ms."
   []
-  (->> (queries)
-       (map second)
-       (apply +)))
+  (let [qs (queries)]
+   [(count qs) (->> qs (map second) (apply +))]))
```

Comment:
A map here will be more understandable since those two numbers can be really similar, like `{:count :total-ms}`

Reply:
yes, updated to a map with 2 keys: `total-queries, total-execution-time-ms`